SimPEG/utils/code_utils.py

```diff
@@ -325,16 +326,32 @@ def __init__(self, add_pckg=None, ncol=3, text_width=80, sort=False):
         core = [
             "SimPEG",
             "discretize",
-            "pymatsolver",
+            "pymatsolver",  # TODO: Will be replaced by pydiso
```

Comment:
It won't be replaced by pydiso. pydiso is specifically a wrapper for intel's pardiso libraries. pymatsolver is another level above that provide abstract operations with solver objects.

Reply:
Oh, thanks for the clarification. Should pydiso be another optional or mandatory package then?